driver/ppm_events_public.h

```diff
@@ -433,6 +433,14 @@ along with sysdig.  If not, see <http://www.gnu.org/licenses/>.
 #define PPM_SETVAL		(1 << 12)
 
 /*
+ * Access flags
+ */
+#define PPM_F_OK            (0)
```

Comment:
Is it OK if I say that  `PPM_F_OK == 1`, `PPM_X_OK == 2`, ... and that it does not strictly correspond to the real value of the flag (e.g., on linux, `F_OK == 0`, `X_OK == 1`, ...)?